src/index_runner/es_indexers/annotated_metagenome_assembly.py

```diff
@@ -10,13 +10,13 @@
 
 _NAMESPACE = "WS"
 _VER_NAMESPACE = "WSVER"
-_AMA_INDEX_VERSION = 1
-_AMA_FEATURES_INDEX_VERSION = 1
+_AMA_INDEX_VERSION = 2
+_AMA_FEATURES_INDEX_VERSION = 2
 _AMA_INDEX_NAME = "annotated_metagenome_assembly_" + str(_AMA_INDEX_VERSION)
 _AMA_FEATURES_INDEX_NAME = "annotated_metagenome_assembly_features_" + str(_AMA_FEATURES_INDEX_VERSION)
 # version indices
-_VER_AMA_INDEX_VERSION = 1
-_VER_AMA_FEATURES_INDEX_VERSION = 1
+_VER_AMA_INDEX_VERSION = 2
+_VER_AMA_FEATURES_INDEX_VERSION = 2
```

Comment:
are there other changes to this indexer

Reply:
Yeah, first there was: https://github.com/kbase/index_runner_spec/pull/14/files
Then I added https://github.com/kbase/index_runner_spec/pull/15/files